_data/chains/eip155-2151.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "BOSAGORA BizNet",
+  "chain": "ETH",
+  "network": "mainnet",
```

Comment:
```suggestion
```

Reply:
Please remove the `network` param as of our latest docs and also run this command locally to format the JSON properly.

`npx prettier --write --ignore-unknown _data`